project.clj

```diff
@@ -30,8 +30,6 @@
                  [io.opentracing.contrib/opentracing-rabbitmq-client "0.1.11" :exclusions [com.rabbitmq/amqp-client]]
                  [org.apache.httpcomponents/fluent-hc "4.5.13"]
                  [org.apache.kafka/kafka-streams "2.8.0" :exclusions [org.slf4j/slf4j-log4j12 log4j]]
-                 [org.apache.logging.log4j/log4j-core "2.14.1"]
```

Comment:
Why are we removing these? Both loggings should be behind feature toggle initially.

Reply:
We have text and json logging behind feature flag. Having both implementations present was causing none of them to be picked for some reason (Didn't debug further). Hence removed log4j altogether.

Reply:
> We have text and json logging behind feature flag
What do you mean? Text logs are supported how? Will this not be backward incompatible?

Reply:
Reason why we cannot have multiple slf4j implementations
```
SLF4J API is designed to bind with one and only one underlying logging framework at a time. If more than one binding is present on the class path, SLF4J will emit a warning, listing the location of those bindings.

When multiple bindings are available on the class path, select one and only one binding you wish to use, and remove the other bindings. For example, if you have both slf4j-simple-2.0.0-alpha5.jar and slf4j-nop-2.0.0-alpha5.jar on the class path and you wish to use the nop (no-operation) binding, then remove slf4j-simple-2.0.0-alpha5.jar from the class path.

The list of locations that SLF4J provides in this warning usually provides sufficient information to identify the dependency transitively pulling in an unwanted SLF4J binding into your project. In your project's pom.xml file, exclude this SLF4J binding when declaring the unscrupulous dependency.
```
[Multiple bindings](http://www.slf4j.org/codes.html#multiple_bindings)

Reply:
> What do you mean? Text logs are supported how? Will this not be backward incompatible?

text logs with MDC are supported via logback itself, for reference please check `logback.test.xml` in this PR